go.mod

```diff
@@ -1,8 +1,9 @@
 module github.com/bitcoin-sv/go-paymail
 
-go 1.21.5
+go 1.22.4
```

Comment:
intentional ? if yes u can resolve this straightaway :)

Reply:
I would prefer if we bump this golang version in separate task.
Or you're completely sure that it won't break anything?

Reply:
Yea - and it has to be bumped to use new models.

Reply:
Done, I went back to the old version